vivisect/tests/testelf.py

```diff
@@ -33,8 +33,27 @@ def do_analyze(vw):
             import traceback
             logging.warning("ERROR in analysis module: (%r): %r", mod, e, exc_info=1)
 
+def cmpnames(x, y):
+    '''
+    Names comparator.  Skips "_#" duplicate name parts so that processing order
+    doesn't matter.
+    '''
+    nmset = [x, y]
+
+    for nmidx in range(2):
+        va, name = nmset[nmidx]
+        if len(name) <= 2:
+            continue
+
+        if name[-2:] in ('_0', '_1', '_2', '_3', '_4', '_5', '_6', '_7', '_8', '_9'):
+            nmset[nmidx] = va, name[:-2]
+
+    return nmset[0] == nmset[1]
+
+
+
```

Comment:
improve name comparisons, such that names that end with _0 still match those that don't (for functions with the same name but want to be unique).